aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -170,12 +170,14 @@ To make an authenticated request to the web API, a bearer token is required. Pos
+   |       <strong>Auth URL</strong>        | `https://login.microsoftonline.com/<tenant domain name>/oauth2/v2.0/authorize?p=B2C_1_SiUpIn` |  Replace <em>&lt;tenant domain name&gt;</em> with the tenant's domain name. **IMPORTANT**: This URL must have the same domain name as what is found in `AzureAdB2C.Instance` in the web API's *appsettings.json* file. See call-out below for more information.            |
    |       <strong>Client ID</strong>       |                <em>&lt;enter the Postman app's <b>Application ID</b>&gt;</em>                 |                                                                                                                                                                                                                                                                              |
    |         <strong>Scope</strong>         |         `https://<tenant domain name>/<api>/user_impersonation openid offline_access`         | Replace <em>&lt;tenant domain name&gt;</em> with the tenant's domain name. Replace <em>&lt;api&gt;</em> with the App ID URI you gave the web API when you first registered it (in this case, `api`). The pattern for the URL is: <em>https://{tenant}.onmicrosoft.com/{api-id-uri}/{scope name}</em>. |
    |         <strong>State</strong>         |                                 <em>&lt;leave blank&gt;</em>                                  |                                                                                                                                                                                                                                                                              |
    | <strong>Client Authentication</strong> |                                Send client credentials in body                                |                                                                                                                                                                                                                                                                              |
 
+    > [!NOTE]
+    > The **Auth URL** can be confusing. The policy settings dialog in the Azure Active Directory B2C portal displays two possible URLs, one in the format `https://login.microsoftonline.com/<tenant domain name>/<additional path information>`, and the other in the format `https://<tenant name>.b2clogin.com/<tenant domain name>/<additional path information>`. It is **critical** that the domain found in in `AzureAdB2C.Instance` in the web API's *appsettings.json* file matches the one used in the web app's *appsettings.json* file. This is the same domain used for the Auth URL field in Postman. Additionally, note that Visual Studio uses a slightly different URL format than what is displayed in the portal. As long as the domains match, the URL will work.
```

Comment:
Couple of nits to consider ...

Perhaps a colon after "URLs" and remove the comma between the two formats?

* "It is" :point_right: "It's"
* matches :point_right: match
* Link Postman to `https://www.getpostman.com/`
* "what is" could be "what's" ... but that's an uncommon contraction.
* "will work" :point_right: "works"

Reply:
Those aren't nits - That's exactly the feedback I wanted :)

Reply:
Pretty sure "matches" is correct, because it is referencing "**the domain** found in in `AzureAdB2C.Instance` in the web API's *appsettings.json* file" (singular).  Also I hesitate to link Postman again because it's linked in the beginning as a prereq.

I just committed everything else.